pod/perlclass.pod

```diff
@@ -227,6 +227,30 @@ If there is no defaulting expression then the parameter is required by the
 constructor; the caller must pass it or an exception is thrown. With a
 defaulting expression this becomes optional.
 
+=head3 :reader
+
+A field with a C<:reader> attribute will generate a reader accessor method
+automatically. By default accessor method will have the same name as the field
+(minus the leading sigil), but a different name can be specified in the
+attribute. The generated method will have an empty (i.e. zero-argument)
```

Comment:
> but a different name can be specified in the attribute
How? If by `:reader(NAME)` let's document it, please.

Reply:
Example added to docs.